include/llarp/dnsc.hpp

```diff
@@ -32,17 +32,21 @@ typedef void (*dnsc_answer_hook_func)(dnsc_answer_request *request);
 struct dnsc_answer_request
 {
   /// sock type
-  void *sock;  // pts to udp...
+  void *sock;  // points to udp that sent the request to DNSc...
   /// customizable (used for hook (outer request))
   void *user;
-  /// storage
+  /// request storage
   dns_msg_question question;
+  /// response storage
+  dns_packet packet;
   /// hook
   dnsc_answer_hook_func resolved;
   /// result
   bool found;
-  llarp::huint32_t result;
-  std::string revDNS;
+
+  // llarp::huint32_t result;
+  // std::string revDNS;
```

Comment:
commented out code?

Reply:
Never heard a good argument for not including it (beyond the muh OCD), I find older comment snippets invaluable, especially when debugging.